test/framework/systemtools.py

```diff
@@ -1052,6 +1052,19 @@ def test_locate_solib(self):
                 self.assertEqual(os.path.basename(libc_path), libname)
                 self.assertTrue(os.path.exists(libc_path), "%s should exist" % libname)
 
+    def test_find_library_path(self):
+        """Test find_library_path function (Linux and Darwin only)."""
+        if get_os_type() == LINUX:
+            libname = 'libc.so.6'
+        elif get_os_type() == DARWIN:
+            libname = 'libSystem.dylib'
+        else:
+            libname = None
+
+        if libname:
+            lib_path = find_library_path(libname)
+            self.assertEqual(os.path.basename(lib_path), libname)
+            self.assertTrue(os.path.exists(lib_path), "%s should exist" % libname)
 
```

Comment:
@lexming Style check is failing, there should be two empty lines above `def suite`... 🙄

Reply:
fixed it myself :)